dpdata/plugins/ase.py

```diff
@@ -1,8 +1,46 @@
 from dpdata.format import Format
+import numpy as np
+try:
+    import ase.io
+except:
```

Comment:
[Do not use base `except:` clause.](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)